README.rst

```diff
@@ -24,80 +24,86 @@
 .. _MNE: https://mne-tools.github.io
 
 `MNE-Python <http://mne-tools.github.io/>`_
-=======================================================
+===========================================
 
-This package is designed for sensor- and source-space analysis of [M/E]EG
-data, including frequency-domain and time-frequency analyses, MVPA/decoding
-and non-parametric statistics. This package generally evolves quickly and
-user contributions can easily be incorporated thanks to the open
-development environment .
+MNE-Python is an open-source Python package for exploring, visualizing, and
+analyzing human neurophysiological data such as MEG, EEG, sEEG, ECoG, and more.
+It includes modules for data input/output, preprocessing, visualization, source
+estimation, time-frequency analysis, connectivity analysis, machine learning,
+and statistics.
 
-Get more information
-^^^^^^^^^^^^^^^^^^^^
 
-If you're unfamiliar with MNE, you can visit the
-`MNE homepage <http://mne-tools.github.io/>`_ for full user documentation.
+Documentation
+^^^^^^^^^^^^^
+
+Documentation for MNE-Python is available at the
+`MNE homepage <http://mne-tools.github.io/>`_.
```

Comment:
please use:

http://martinos.org/mne/stable/index.html

as home address

Reply:
I'd prefer `https://martinos.org/mne` as it's shorter